src/Microsoft.OData.Edm/Microsoft.OData.Edm.csproj

```diff
@@ -85,4 +85,19 @@
       <SubType>Designer</SubType>
     </EmbeddedResource>
   </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Text.Json">
+      <Version>4.7.1</Version>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>

```

Comment:
What is this include?  Why is it added twice (here and on ln 89-91)?

Reply:
That's from Visual Studio automatically. I didn't add it. Will remove it

---
In reply to: [429366353](https://github.com/OData/odata.net/pull/1775#discussion_r429366353) [](ancestors = 429366353)